niworkflows/interfaces/nibabel.py

```diff
@@ -186,6 +186,7 @@ class _MergeSeriesInputSpec(BaseInterfaceInputSpec):
     allow_4D = traits.Bool(
         True, usedefault=True, desc="whether 4D images are allowed to be concatenated"
     )
+    affine_tolerance = traits.Float(desc="Absolute tolerance allowed between image affines")
```

Comment:
What about setting a default (taken from numpy), and doing our own check?

```suggestion
    affine_tolerance = traits.Float(1e-08, usedefault=True, desc="Absolute tolerance allowed between image affines")
```